crates/factor-outbound-http/src/wasi.rs

```diff
@@ -49,7 +49,10 @@ where
     wasmtime_wasi_http::bindings::http::outgoing_handler::add_to_linker::<_, HasHttp>(
         linker, get_http,
     )?;
-    wasmtime_wasi_http::bindings::http::types::add_to_linker::<_, HasHttp>(linker, get_http)?;
+    let link_opts = wasmtime_wasi_http::bindings::http::types::LinkOptions::default();
+    wasmtime_wasi_http::bindings::http::types::add_to_linker::<_, HasHttp>(
+        linker, &link_opts, get_http,
```

Comment:
Does it work to say `&Default::default()` for the second arg, and remove the explicit `link_opts` object?  I'm not familiar enough to be sure, but if it does it might make intent clearer?